src/components/Calculator.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+function CalculatorKey(props) {
+  const props1 = { ...props };
+  return (<input type="button" className={(Number(props1.value) > 0) ? 'number' : 'key'} value={props1.value} />);
+}
+
+const resultElement = <input type="number" className="result" value="0" disabled />;
+
+class Calculator extends React.Component {
+  keys = ['AC', '+/-', '%', '÷',
+    7, 8, 9, 'x',
+    4, 5, 6, '-',
+    1, 2, 3, '+',
+    0, '.', '='];
+
+  keysElements = this.keys.map((key) => (<CalculatorKey value={key} key={key.toString()} />));
+
```

Comment:
- [ ] Great job so far 👏🏻 But, According to the project requirements, The Calculator.js component should be a class-based component and not a stateless function component. Your code might look like this 👇🏻 


```javascript
// ./src/components/Calculator.js

import React, { Component } from 'react';

class Calculator extends Component {
  render() {
    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}

export default Calculator;
```

- Kindly make this 👍🏻